app/assets/stylesheets/forever_style_guide/modules/_button.scss

```diff
@@ -47,38 +49,17 @@
       @include btn-color('gray');
     }
   }
-  &.btn-primary,
-  &.btn-success {
-    @include btn-color('primary');
-    &.btn-inverse {
-      @include btn-color('primary', true);
-    }
-  }
-  &.btn-secondary,
-  &.btn-info {
-    @include btn-color('secondary');
-    &.btn-inverse {
-      @include btn-color('secondary', true);
-    }
-  }
-  &.btn-accent {
-    @include btn-color('accent');
-    &.btn-inverse {
-      @include btn-color('accent', true);
-    }
-  }
-  &.btn-warning {
-    @include btn-color('warning');
-    &.btn-inverse {
-      @include btn-color('warning', true);
-    }
-  }
-  &.btn-danger {
-    @include btn-color('danger');
-    &.btn-inverse {
-      @include btn-color('danger', true);
+
+  // creates a button style for each color in color dictionary
+  @each $id, $color in $colors {
```

Comment:
Here's a sample of the errors I'm encountering when running this against the store. It looks like somewhere between this change, the semantic colors and the color_utils, something's not quite right.

<img width="1011" alt="screen shot 2015-09-08 at 9 17 39 am" src="https://cloud.githubusercontent.com/assets/3814998/9736854/a8ce8924-5611-11e5-9849-b2218ff7a589.png">